src/clnt_vc.c

```diff
@@ -312,7 +312,7 @@ clnt_vc_ncreate2(int fd,	/* open file descriptor */
 		if (ct->ct_addr.len)
 			mem_free(ct->ct_addr.buf, ct->ct_addr.len);
 
-		if (xd->refcnt == 1) {
```

Comment:
I don't see where xd->hdl.xd is set to xd in this function?  So in the create case, I don't think it will be decremented, because it will be NULL.